products/memgraph/upcoming/tutorials/football-transfers.md

```diff
@@ -0,0 +1,274 @@
+
+```opencypher
+MATCH (m:Team )-[:TRANSFERRED_FROM]->(t:Transfer)<-[:TRANSFERRED_IN]-(p:Player), (t)-[:TRANSFERRED_TO]->(n:Team)
+WHERE m.name="FC Barcelona"
+RETURN n.name as team_name, collect(p.name) as names, COUNT(p) AS number_of_players
+ORDER BY number_of_players DESC
+```
+
+
+7) What are the teams that most players went to in season 2003/2004? The results may surprise you. 
+
+```opencypher 
+MATCH (s:Season {name:"2003/2004"})<-[:HAPPENED_IN]-(t:Transfer)<-[:TRANSFERRED_IN]-(player:Player)
+MATCH (t)-[:TRANSFERRED_TO]->(n:Team)
+WITH DISTINCT player, n
+RETURN n.name as team_name, COUNT(player) AS number_of_players,collect(player) as names
+ORDER BY number_of_players DESC, team_name
+LIMIT 20;
+```
+
+8)In great teams, there are usually players who seem to be irreplaceable. When they leave,
+the board is often struggling to find a proper replacement for them. That's why there are more
+transfers on those positions. Let's find out on which positions in season
+2015/2016 FC Barcelona spent money.
+
+```opencypher 
+MATCH (n:Team )-[:TRANSFERRED_FROM ]->(t:Transfer)<-[:TRANSFERRED_IN]-(p:Player ),
+(s:Season)<-[:HAPPENED_IN]-(t)-[r1:TRANSFERRED_TO]->(m:Team)
+WHERE t.fee IS NOT NULL AND (s.name IN ["2015/2016","2016/2017"]) AND m.name="FC Barcelona"
+RETURN collect( p.name) AS names, p.position AS position, SUM(t.fee) AS money_spent_per_position
+```
+
+9) But what was the most money FC Barcelona spent per position from 1992/1993 season to 2019/2020?
+
+```opencypher 
+MATCH (a:Team )-[:TRANSFERRED_FROM ]->(t:Transfer)<-[:TRANSFERRED_IN ]-(p:Player ), (t)-[:TRANSFERRED_TO]->(b:Team)
+WHERE t.fee IS NOT NULL AND b.name="FC Barcelona"
+WITH max(t.fee) as maxMoneySpent, p.position AS position
+RETURN maxMoneySpent, position
+```
+
+10) Now, let us find who were the most expensive players per position in our team FC Barcelona. 
+
+```opencypher 
+MATCH
+    (barca:Team)<-[:TRANSFERRED_TO]-(t:Transfer)<-[:TRANSFERRED_IN]-(p:Player), (t)-[:HAPPENED_IN]->(s:Season)
+WHERE 
+    t.fee is NOT NULL AND 
+    barca.name = "FC Barcelona"
+WITH p.position as player_position, max(t.fee) as max_fee
+
+MATCH
+    (p:Player)-[:TRANSFERRED_IN]->(t:Transfer)-[:HAPPENED_IN]->(s:Season)
+WHERE
+    p.position = player_position AND
+    t.fee = max_fee 
+RETURN
+    max_fee, player_position, collect(p.name) as player_names
+```
+
+
+11) If you want to find indirect transfers of players between two clubs you can do that also
+
+```opencypher 
+MATCH
+    (t:Transfer)<-[e:TRANSFERRED_IN]-(player:Player)-[:TRANSFERRED_IN]->(:Transfer)<-[:TRANSFERRED_FROM]-(team:Team)
+WHERE
+    team.name = "FC Barcelona"
+WITH player, collect(t) as transfers, collect(e) as connections
+MATCH
+  path_indirect = (a:Team)-[*bfs..10 (e, n | 'Team' IN labels(n) OR ('Transfer' in labels(n) AND n in transfers) )]->(b:Team)
+WHERE
+  a.name = "FC Barcelona" AND
+  b.name = "Sevilla FC"
+UNWIND connections as connection
+RETURN player, connection, path_indirect
```

Comment:
Also, as noted before in previous PR comments, returning `path` and `connection` doesn't have any benefits to someone running this in mgconsole or any other terminal tool. So, add a description below where you can say that if you are running this in Lab with visuals, you can change query in some way to get full graph of two teams, transfers and players.

Reply:
:+1: